packages/less/src/less-node/file-manager.js

```diff
@@ -70,13 +70,20 @@ class FileManager extends AbstractFileManager {
                                 fullFilename = path.join(paths[i], fullFilename);
                             }
 
-                            if (!explicit && paths[i] === '.') {
+                            const isNodeModulePath = !!paths[i].match(/[/\\]node_modules[/\\]?/g);
+
+                            if (!explicit && (paths[i] === '.' || isNodeModulePath)) {
```

Comment:
Inline the variable as it does not seem to be used anywhere else
```suggestion
                            if (!explicit && (paths[i] === '.' || /*isNodeModulePath*/ Boolean(paths[i].match(/[/\\]node_modules[/\\]?/g)))) {
```